db/seeds.rb

```diff
@@ -5,3 +5,51 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+
+puts "Cleaning database"
+Cat.destroy_all
+User.destroy_all
+
+puts "Seeding users..."
+
+10.times do
+  user = User.create(
+    email: "Faker::Internet.email",
+    password: "secret",
+    username: ""
+    )
+end
+
+puts "Users created."
+
+puts "Seeding cats..."
+
+30.times do
+  cat = Cat.create(
+    name: "Faker::Name.first_name",
+    breed: "Faker::Creature::Cat.breed",
+    color: "Faker::Color.color_name",
+    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat...",
+    user_id: rand(1..30)
```

Comment:
we have only 10 users created up, so could you please either create user 30.times or change the rand(1..10)?

Reply:
see change in the next commit, sorry